### PR TITLE
tranfer: document ops and dev transfer

### PR DESCRIPTION
**Pull request by vbatts**
Facilitating the "how do i do X?" knowledge transfer process.

Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

File: transfer.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+## Equivalents
+
+For many troubleshooting and information collection steps, there may be an existing pattern.
+Following provides equivalent with cri-o tools for gathering information or jumping into containers, for operational use.
+
+| Existing Step | CRI-O (and friends) |
+| :---: | :---: |
+| `docker info` | [`kpod info`](./docs/kpod-info.2.md) |
+| `docker inspect` | [`kpod inspect`]() (TBD) |
+| `docker stats` | [`crioctl ctr status`]() (TBD) |
```
Comment (mheon) [inline transfer.md]: Should we recommend the kpod versions of these? We have a `kpod stats` PR which I do not think is merged, for example

Comment (vbatts) [inline transfer.md]: oh, if there is a kpod subcommand, we can use it. That was not in the works when i started this doc.

**Comment by mrunalp** [comment]
@vbatts git validation failed with whitespace errors.

**Comment by vbatts** [comment]
updated

File: transfer.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# cri-o Usage Transfer
+
+This document outlines useful information for ops and dev transfer as it relates to infrastructure that utilizes cri-o.
+
+## Operational Transfer
+
+## Abstract
+
+The `crio` daemon is intended to provide the [CRI](https://github.com/kubernetes/community/blob/master/contributors/devel/container-runtime-interface.md) socket needed for Kubernetes to use (See the document for [configuring kubernetes to use cri-o](./kubernetes.md) for more information on that).
```
Comment (TomSweeneyRedHat) [inline transfer.md]: suggest "... Kubernetes to use for automating deployment, scaling, and management of containerized applications (See the document...)" or a few other words of your choosing to explain how Kubernetes uses the daemon.  Or perhaps just change "Kubernetes to use" to "Kubernetes operations".

File: transfer.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Therefore the `crioctl` command line is a client that interfaces to the same grpc socket as the kubernetes daemon would, for talking to the `crio` daemon.
+In many ways `crioctl` is only as feature rich as the Kubernetes CRI requires.
+For this there are additional tools e.g. `kpod` particularly for human interaction.
```
Comment (TomSweeneyRedHat) [inline transfer.md]: suggest dropping "For this" and maybe going with something like:  "There are additional tools e.g. `kpod` and `buildah` that provide a feature rich set of commands for all operational needs in a Kubernetes environment."  Or some such.  Given buildah is mentioned below, I think we should work that in here too in what ever you decide to go with.

File: transfer.md (hunk 2 above)
Comment (vbatts) [inline transfer.md]: done

File: transfer.md (hunk 3 above)
Comment (vbatts) [inline transfer.md]: done

**Comment by vbatts** [comment]
updated

**Comment by rhatdan** [comment]
LGTM.
Mrunal Lets merge this, since tests are not necessary.

**Comment by TomSweeneyRedHat** [comment]
LGTM, thanks for the touch ups.

**Comment by runcom** [comment]
LGTM

**Pull request merged**